### PR TITLE
oslo.context: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/oslo.context/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pbr >=2.0.0
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - pbr >=2.0.0
```
Comment (nehaljwani) [inline recipes/oslo.context/meta.yaml:24]: The wheel has:
```
Requires-Python: >=3.6
Requires-Dist: pbr (!=2.1.0,>=2.0.0)
Requires-Dist: debtcollector (>=1.2.0)
```
We should probably also add the constraint on version 2.1.0 for pbr.
xref: https://github.com/openstack/oslo.context/blob/3.0.2/requirements.txt#L5

Comment (tschoonj) [reply recipes/oslo.context/meta.yaml:24]: Fixed!

File: recipes/oslo.context/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pbr !=2.1.0,>=2.0.0
+    - pip
+    - python >=3.6
+  run:
+    - python !=2.1.0,>=3.6
```
Comment (nehaljwani) [inline recipes/oslo.context/meta.yaml:23]: This constraint was supposed to be on pbr :-)

Comment (tschoonj) [reply recipes/oslo.context/meta.yaml]: Good catch! Fixed...

**Comment by nehaljwani** [comment]
Thanks @tschoonj !

**Comment by tschoonj** [comment]
Thank you! I will be adding a couple more later today...

**Pull request merged**